### PR TITLE
deps: bls: downgrade bls library

### PR DESCRIPTION
**Pull request by jennijuju**
fix https://github.com/filecoin-project/lotus/issues/12467 for v1.28.x

### CODE REVIEW

**Review by rvagg — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Lotus changelog
 
+# 1.28.3 / 2024-09-16
+
+This patch fixes the invalid bls signature bug that is causing many of the lotus node to fail in syncing.
```
Comment (BigLep) [inline CHANGELOG.md:5]:
```suggestion
This patch fixes the invalid bls signature bug that is causing many lotus nodes to fail in syncing (see [#12467](https://github.com/filecoin-project/lotus/issues/12467)).
```

File: go.mod [hunk 2]
```diff
@@ -12,6 +12,8 @@ replace github.com/filecoin-project/test-vectors => ./extern/test-vectors // pro
 
 replace github.com/filecoin-project/filecoin-ffi => ./extern/filecoin-ffi // provided via a git submodule
 
+replace github.com/kilic/bls12-381 v0.1.1-0.20220929213557-ca162e8a70f4 => github.com/kilic/bls12-381 v0.1.0 // to fix bls signature validation
```
Comment (BigLep) [inline go.mod:15]:
```suggestion
replace github.com/kilic/bls12-381 v0.1.1-0.20220929213557-ca162e8a70f4 => github.com/kilic/bls12-381 v0.1.0 // to fix bls signature validation issue from https://github.com/filecoin-project/lotus/issues/12467
```

**Review by BigLep — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+
+
```
Comment (rvagg) [inline CHANGELOG.md:7]:
```suggestion
This patch fixes the invalid bls signature bug that is causing many of the lotus node to fail in syncing. Fixes [filecoin-project/lotus#12467](https://github.com/filecoin-project/lotus/pull/12467).

```

**Comment by rjan90** [comment]
Was the plan to bump the version in a seperate PR?

**Pull request merged**